launch/test_param_srv.launch

```diff
@@ -39,14 +42,16 @@
                            RTK_data_transmission_interruption_limit: 5,
                            RTK_correction_protocol: "RTCM3",
                             <!-- # TCP - Inertial Sense, -->
-                            RTK_server_IP: "127.0.0.1",
+                            RTK_server_IP: "66.219.246.93",
```

Comment:
We can leave this only if we don't think customers will be running the test.  We don't want people to be using/loading our RTK base station.

Reply:
Good point. I planned to revert but forgot.